app/presenters/keyset_pagination_presenter.rb

```diff
@@ -0,0 +1,40 @@
+module Presenters
+  class KeysetPaginationPresenter
+    def initialize(pagination, request_url)
+      @results = pagination.call.as_json
+      @pagination = pagination
+      @request_url = request_url.to_s
+    end
+
+    def present
+      {
+        links: links,
+        results: results
+      }
+    end
+
+  private
+
+    attr_reader :results, :pagination, :request_url, :present_record_filter
+
+    def links
+      [next_link]
```

Comment:
Could we have self and previous links? 

fun challenge here: is can we know if there are actually next or previous items?